src/OpenTelemetry.Instrumentation.Http/HttpClientMetricInstrumentationOptions.cs

```diff
@@ -0,0 +1,42 @@
+// <copyright file="HttpClientMetricInstrumentationOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+using Microsoft.Extensions.Configuration;
+using static OpenTelemetry.Internal.HttpSemanticConventionHelper;
+
+namespace OpenTelemetry.Instrumentation.Http
+{
+    internal class HttpClientMetricInstrumentationOptions
```

Comment:
Why are we adding this class?

Is it only required to support Configuration providers other than environment variables?

Reply:
Ya. We could probably do it without the class if you really don't want it. But I think this makes it clean and consistent with all the other spots doing the same.

Reply:
I think it should be fine considering this is an `internal` class.